code/modules/antagonists/blob/blob_abilities.dm

```diff
@@ -248,14 +247,14 @@
 
 		if (!owner.tutorial)
 			//do a little "blobsplosion"
+		//sound and sound code additions by retkid#9135, coding assitance by pali#0439 and Paul#7955
 			var/amount = rand(20, 30)
 			src.auto_spread(startTurf, maxRange = 3, maxTurfs = amount)
-
+		owner.playsound_local(owner.loc, 'sound/voice/blob/blobdeploy1.ogg', 50, 1)
```

Comment:
why blobdeploy1 if there's only 1 at all :flooshed:

Reply:
Because there was a blob 2 at some point and I forgot to rename